composeApp/src/commonMain/kotlin/com/overdrive/cruiser/views/MySpotsView.kt

```diff
@@ -57,134 +75,264 @@ fun MySpotsView(mySpotsViewModel: MySpotsViewModel, onAddSpotClick: () -> Unit)
+                                    backgroundColor = Color(
+                                        0xFFF9784B
+                                    )
+                                ),
+                                modifier = Modifier
+                                    .fillMaxWidth()
+                                    .height(55.dp)
+                                    .clip(RoundedCornerShape(12.dp))
                             ) {
-                                Text("226 total bookings")
+                                Text(
+                                    text = "Edit Spot",
+                                    color = Color.White
+                                )
                             }
-                        }
-                        Button(
-                            onClick = { onAddSpotClick() },
-                            shape = RoundedCornerShape(12.dp),
-                            colors = ButtonDefaults.buttonColors(backgroundColor = Color(0xFFF9784B)),
-                            modifier = Modifier
-                                .fillMaxWidth()
-                                .height(55.dp)
-                                .clip(RoundedCornerShape(12.dp))
-                        ) {
-                            Text(
-                                text = "Edit Spot",
-                                color = Color.White
-                            )
-                        }
 
-                        Spacer(modifier = Modifier.weight(1f))
+                            Spacer(modifier = Modifier.height(8.dp))
+                            Button(
+                                onClick = {
+                                    scope.launch {
+                                        showDialog = true
+                                    }
+                                },
+                                shape = RoundedCornerShape(12.dp),
+                                colors = ButtonDefaults.buttonColors(
+                                    backgroundColor = Color.White,
+                                ),
+                                modifier = Modifier
+                                    .fillMaxWidth()
+                                    .height(55.dp)
+                                    .clip(RoundedCornerShape(12.dp))
+                                    .border(1.dp, Color(0xFFFF0000), RoundedCornerShape(12.dp))
+                            ) {
+                                Text(
+                                    text = "Delete Spot",
+                                    color = Color(0xFFFF0000),
+                                )
+                            }
 
-                        Button(
-                            onClick = {
-                                scope.launch {
-                                    showDialog = true
+                            Spacer(modifier = Modifier.height(16.dp))
```

Comment:
Spacer here is further than the previous one, should probably make them the same size of separate them further